app/assets/javascripts/components/profile_card.es6.jsx

```diff
@@ -1,12 +1,26 @@
 class ProfileCard extends React.Component {
 
+  static getpropTypes() {
+    return {
+      cardName: React.PropTypes.string,
+      cardBody: React.PropTypes.string,
```

Comment:
We should add `.isRequired` after each `React.PropTypes.string` since these props will be required for these cards.